recipes/pydivert/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "pydivert" %}
+{% set version = "2.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pydivert-{{ version }}.tar.gz
+  sha256: f0e150f4ff591b78e35f514e319561dadff7f24a82186a171dd4d465483de5b4
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: True  # [not win]
```

Comment:
Just curious, what is the issue that limits this on Windows?

Reply:
pydivert is a Windows-only package